.github/README.md

```diff
@@ -39,6 +40,81 @@ cd  # in your home directory
 git clone https://github.com/torvalds/linux.git
 ```
 
+If you are compiling the kernel for the first time, install some of the packages required:
+
+On a Debian/Ubuntu distribution:
+
+
+```
+sudo apt install -y \
+  qtdeclarative5-dev \
+  pkg-config \
+  bison \
+  flex \
+  libelf-dev \
+  libssl-dev \
+  libcap-dev \
+  llvm
+```
+
+Enable the folliwing config options to ensure for `hawk` to work:
```

Comment:
following*
"to ensure that hawk works"* or "for hawk to work"

Reply:
Done.